exploration/docs/terminology.tex

```diff
@@ -127,8 +127,8 @@ \section{Optical Constructions}
 may be written explicitly as
 
 \[
-l_{12}(a) := (x_1 \otimes y_1, y_2) \text { where } x = l_1(a), y = l_2(x_2) \]\[
```

Comment:
Ok. The subscript was supposed to indicate indexing, but I agree this is cleaner when compared to the rest of the document.

Reply:
`y_2` should be `c` then in this convention